runtime/near-vm-runner/src/near_vm_runner/metrics.rs

```diff
@@ -0,0 +1,19 @@
+use near_o11y::metrics::{try_create_int_counter_vec, IntCounterVec};
+use once_cell::sync::Lazy;
+
+pub static COMPILED_CONTRACT_CACHE_HIT: Lazy<IntCounterVec> = Lazy::new(|| {
+    try_create_int_counter_vec(
+        "near_compiled_contract_cache_hit",
```

Comment:
Please follow the [official best practices on metric naming](https://prometheus.io/docs/practices/naming/). It hasn't been followed very much so far with metrics added previously, but I don't see a reason for us to continue being lax with this.

In particular counters should be `_total` for unitless counter metrics.

Reply:
done, please take another look.